src/main/java/io/cryostat/net/web/http/api/v1/ReportGetHandler.java

```diff
@@ -122,15 +122,31 @@ public void handleAuthenticated(RoutingContext ctx) throws Exception {
         String recordingName = ctx.pathParam("recordingName");
         List<String> queriedFilter = ctx.queryParam("filter");
         String rawFilter = queriedFilter.isEmpty() ? "" : queriedFilter.get(0);
-        try {
 
-            Path report =
-                    reportService
-                            .get(recordingName, rawFilter)
-                            .get(reportGenerationTimeoutSeconds, TimeUnit.SECONDS);
-            ctx.response().putHeader(HttpHeaders.CONTENT_TYPE, HttpMimeType.HTML.mime());
-            ctx.response()
-                    .putHeader(HttpHeaders.CONTENT_LENGTH, Long.toString(report.toFile().length()));
+        String accept = ctx.request().headers().get(HttpHeaders.ACCEPT);
+        accept = accept == HttpMimeType.UNKNOWN.mime() ? HttpMimeType.HTML.mime() : accept;
```

Comment:
`==` should be `.equals()`. Is this ternary really needed though? Seems like the `switch` below should handle it.

Reply:
I was trying to remember why I was doing that, and then I remembered it's because if you feed in a header like:
`
$ curl ... -H "Accept: applica" 
`
It's going to correctly give 
```
Jun 28, 2022 5:41:04 PM io.cryostat.core.log.Logger warn
WARNING: HTTP 406: Not Acceptable
```

But if you give whitespace as an Accept header like  
`
$ curl ... -H "Accept:      " 
`
It will set `accept = null` and `switch(null)` doesn't seem to valid in Java.

I guess it would be easier to just do 
```
if (accept == null) {
    throw new HttpException(statusCode: 406)
}
// or do whatever when the header is null
```

Reply:
Yea, you could add a null/isBlank guard before the switch and just throw early. I think that would be more readable. Then your `switch` and everything after only has to deal with non-blank inputs.